content/en/xtc-release-notes/v34.md

```diff
@@ -0,0 +1,23 @@
+---
+title: XTC v34
+linkTitle: XTC v34
+
+weight: 183
+type: docs
+
+date: 2021-05-26
+
+description: >
+---
+
+## XTC v34
+
+
+### Load Testing
+- Load agent machines now use XLT [XLT 5.4.1]({{< relref "/release-notes/5.4.x.md" >}}).
```

Comment:
double "XLT" here, please fix

Reply:
Also, the links are not valid any longer after your merge of master (which is kinda intentional). The version pages of XLT are now named e.g. 5_4_x.md (underscores instead of dots in filename), please change all links accordingly. You can see which files are affected on building, there should be warnings on the console about "REF_NOT_FOUND":

![Screenshot from 2022-02-22 16-44-05](https://user-images.githubusercontent.com/51991513/155167483-1cd1a746-de8a-4346-89b2-70b35c199ab1.png)